teos/src/api/tor.rs

```diff
@@ -1,16 +1,44 @@
+use std::convert::TryInto;
 use std::io::{Error, ErrorKind};
 use std::net::SocketAddr;
+use std::path::PathBuf;
+
+use tokio::fs;
 use tokio::net::TcpStream;
 use tokio::time::{sleep, Duration};
 use torut::control::UnauthenticatedConn;
 use torut::onion::TorSecretKeyV3;
-use triggered::Listener;
+use triggered::{Listener, Trigger};
+
+/// Loads a Tor key from disk (if found).
+async fn load_tor_key(path: PathBuf) -> Option<TorSecretKeyV3> {
+    log::info!("Loading Tor secret key from disk");
+    let key = fs::read(path.join("onion_v3_sk"))
```

Comment:
Should we handle the `ERROR [teos::api::tor] Cannot load Tor secret key. No such file or directory (os error 2)` on the first run? When someone first kicks this off they will always see that error message when Tor is configured since the file will not have been created yet. I can see just leaving it as is though, as the program will continue & create a new file even after reporting that error.

Reply:
I think it makes sense not to Err here, it feels like bad UX.

Will fix this.

Reply:
> Should we handle the ERROR [teos::api::tor] Cannot load Tor secret key. No such file or directory (os error 2) on the first run? 

Yeah, feels more of a warning thing.